service.onmute/addon.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.onmute" version="0.0.1" name="OnMute" provider-name="Lunatixz">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.simplejson" version="3.3.0"/>
```

Comment:
afaik json **is** simplejson and json is packed with pythons stdlibs.
simplejson however is updated more frequently and apparently way faster, so unless you really need speed (at which point cjson might by of interest too), just stick with pythons default json module.

You might see things like...
```
try:
    import simplejson as json
except ImportError:
    import json
```
...where they basically use simplejson if available but fall back to json in case it isn't.
Again, unless you **know** you need the benefits of simplejson, there's no real reason to use it